Source/Core/Cartographic.js

```diff
@@ -146,6 +148,24 @@ define([
         return result;
     };
 
+    /**
+     * Returns a Cartesian3 position from a {@link Cartographic} input.
+     *
+     * @param {Cartographic} Cartographic input to be converted into a Cartesian3 output.
+     * @param {Ellipsoid} [ellipsoid=Ellipsoid.WGS84] The ellipsoid on which the position lies.
+     * @param {Cartesian3} [result] The object onto which to store the result.
+     * @returns {Cartesian3} The position
+     */
+    Cartographic.toCartesian3 = function(cartographic, ellipsoid, result) {
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(cartographic)) {
```

Comment:
This block can be replaced with `Check.defined('cartographic', cartographic)`, which is out new way of doing parameter checking.